src/ng-widget.coffee

```diff
@@ -50,7 +50,13 @@ angular.module 'ngWidget', []
           @scope[key] = '=' if value is 'two-way'
           @scope[key] = '&' if value is 'function'
 
+    # Alias for the link function
+    @ready = (callback)->
+      @link = callback
```

Comment:
this overrides the link completely right? This would break our `widget.on()` shit